src/app/parcels/ActionBar/Actions.tsx

```diff
@@ -93,6 +98,19 @@ const Actions: React.FC<Props> = ({
 }) => {
     const [selectedParcels, setSelectedParcels] = useState<ParcelsTableRow[]>([]);
     const [modalToDisplay, setModalToDisplay] = useState<ActionName | null>(null);
+    const [availableActionsForUserRole, setAvailableActionsForUserRole] = useState<ActionTypes[]>(
+        []
+    );
+    const { role } = useContext(RoleUpdateContext);
+
+    useEffect(() => {
+        if (role === "volunteer") {
```

Comment:
Out of interest, why are we singling out volunteers. The ticket name is "Display action list including delete parcel if user is admin" which would imply that only admins can delete parcels. What about staff / managers or the other roles - what should they have access to?

Reply:
To make this clearer, we could change availableToVolunteers to rolesAvailableTo = ["admin","staff"...]. I would have thought that would simplify the code too? Lmk if that doesn't work.

Reply:
Yes that's what I initially thought too but Dan commented after I've written the QA that manager, staff and admin should be able to delete parcels too. https://github.com/NorwoodAndBrixtonFoodbank/nbf-website/issues/236#issuecomment-2119949498

I think that is a good suggestion - let me try implementing

Reply:
Ah that makes sense, apologies for not seeing this! Could you update the what's changed in the pr to match this :)